apps/www/components/PoliticsCommunityQuestionnaire/Person.js

```diff
@@ -0,0 +1,245 @@
+          if (error) {
+            if (process.browser) {
+              router.replace({ pathname: OVERVIEW_QUESTIONNAIRE_PATH })
+            }
+            return null
+          }
+          const {
+            questionnaire: { questions, results },
+          } = data
+          const submission = results.nodes[0]
+          if (!submission) {
+            if (process.browser) {
+              router.replace({ pathname: OVERVIEW_QUESTIONNAIRE_PATH })
+            }
+            return null
+          }
+
+          const meta = {
+            url,
+            title: '26 Fragen zur Politik',
+            description: 'Die Antworten von ' + author?.name,
+            image: `${ASSETS_SERVER_BASE_URL}/render?width=1200&height=1&url=${encodeURIComponent(
+              shareImageUrl,
+            )}`,
+          }
+
+          return (
+            <>
+              <Meta data={meta} />
+              <div
+                style={{
+                  backgroundColor: QUESTIONNAIRE_BG_COLOR,
+                  padding: '24px 0 24px',
+                }}
+              >
+                <ColorContextProvider colorSchemeKey='light'>
+                  <div
+                    style={{
+                      paddingTop: 24,
+                      textAlign: 'center',
+                    }}
+                  >
+                    <Figure
+                      size='tiny'
+                      attributes={{ style: { position: 'relative' } }}
+                    >
+                      <FigureImage src={QUESTIONNAIRE_SQUARE_IMG_URL} />
+                      <FigureCaption>
+                        <FigureByline>Cristina Spanò</FigureByline>
+                      </FigureCaption>
+                    </Figure>
+                    <NarrowContainer style={{ padding: '20px 0' }}>
+                      <Interaction.Headline>
+                        {'Die Antworten von ' + author?.name}
+                      </Interaction.Headline>
+                      {author?.profilePicture && (
+                        <img
+                          src={author.profilePicture}
+                          style={{
+                            marginTop: 30,
+                            width: 120,
+                            borderRadius: 80,
+                          }}
+                        />
+                      )}
+                    </NarrowContainer>
+                  </div>
+                </ColorContextProvider>
+              </div>
+              <Center>
+                <div
+                  {...submissionStyles.header}
+                  style={{
+                    top: headerHeight,
+                    padding: '10px 0',
+                    margin: '10px 0 0',
+                  }}
+                  {...colorScheme.set('backgroundColor', 'default')}
+                >
+                  <div {...submissionStyles.headerText}>
+                    <NextLink href={OVERVIEW_QUESTIONNAIRE_PATH} passHref>
```

Comment:
Probably we've done this elsewhere too but is IconButton a `<a>` element? If it's a `<button>` it works but is wrong HTML.

Reply:
hmm. what shall i do then?

Reply:
Nevermind, apparently [IconButton renders automagically as `<a>`](https://styleguide.republik.ch/components/iconbutton) when `href` is passed.